com.unity.render-pipelines.core/Runtime/RenderPipeline/IShaderVariantsSettings.cs

```diff
@@ -0,0 +1,28 @@
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// Specifies the logging level for shader variants
+    /// </summary>
+    public enum ShaderVariantLogLevel
+    {
+        Disabled,
+        OnlySRPShaders,
+        AllShaders,
+    }
+
+    /// <summary>
+    /// Exposes settings for shader variants
+    /// </summary>
+    public interface IShaderVariantsSettings
```

Comment:
I think I would prefer singular here (`IShaderVariantSettings` instead of `VariantsSettings`) but it's not a very strong opinion :)